atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/ReadValidationCommitTimestampLoader.java

```diff
@@ -124,7 +126,8 @@ private ListenableFuture<LongLongMap> getCommitTimestampsForTransactionsStartedA
                             "An uncommitted conflicting read was written after our start timestamp for table "
                                     + tableRef + ".  This case can cause deadlock and is very likely to be a "
                                     + "read write conflict.",
-                            tableRef);
+                            tableRef,
```

Comment:
So tableRef is enough to construct the args. I am assuming we still pass the arg because adding the dep for LoggingArgs is awkward?

Reply:
Couple reasons:
- `atlasdb-api` should not have a dependency on `atlasdb-client`
- We should give callers the flexibility to provide whatever args they want